src/cmd-remote-build

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3 -u
+
+import sys
+import argparse
+from os import environ
+from subprocess import check_call, check_output, CalledProcessError
+
+def run_cmd(command, output=None):
+    '''
+    Run the given command using check_call and verify its return code.
+    @param str command command to be executed
+    '''
+    if output:
+        try:
+            output = check_output(command.split())
+        except CalledProcessError as e:
+            print('An exception has occurred: {0}'.format(e))
+            sys.exit(1)
+        return output
+    else:
+        try:
+            check_call(command.split())
+        except CalledProcessError as e:
+            print('An exception has occurred: {0}'.format(e))
+            sys.exit(1)
+
+def build(arch, commit, file, repo):
+    '''
+    Build the image  using podman remote and push to the registry
+    @param arch
+    @param commit
+    @param file
+    @param repo
+    @param tag
+    '''
+    tag = ("%s-%s" % (arch, commit))
+    run_cmd("podman --remote build --tag %s:%s-%s -f %s" % (repo, arch, commit, file))
```

Comment:
the `-f src/DockerFile` is interesting to me here.

How does this work? Is `src/DockerFile` local and podman somehow exports everything over to the remote before doing the build? Instead of `-f src/Dockerfile` shouldn't we just be passing the full directory as the context?

Reply:
> How does this work? Is src/DockerFile local and podman somehow exports everything over to the remote before doing the build?

Yes, `src/DockerFile` is local, and  it seems export the files the builds will need. 

> Instead of -f src/Dockerfile shouldn't we just be passing the full directory as the context?
The [doc](https://docs.podman.io/en/latest/markdown/podman-build.1.html) says it should be a Containerfile. I guess I tried the directory and got some error around it, but I can double check, I don't recall exactly if it was the case.